### PR TITLE
Added metadata as per new suite addition to run in pipeline

### PR DESCRIPTION
**Pull request by sunilangadi2**
Signed-off-by: Sunil Angadi <sangadi@redhat.com>

# Description
Updated metadata as per new suite addition to run in the pipeline
**new suites**:

- _suites/pacific/cephadm/customer_bz/tier-2_multiple_subnet_openstack_only.yaml_ 
for this test suite tier-2 stage 6 have already 4 suites hence added it as new stage-7

- _suites/pacific/dashboard/tier-1_dashboard_alerts.yaml_ 
for this test suite tier-1 stage 3 have already 4 suites hence added it as new stage-4

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by manasagowri — DISMISSED** [review]
LGTM

**Comment by vamahaja** [comment]
@sunilangadi2 please rebase and resolve the conflicts

File: pipeline/metadata/5.3.yaml [hunk 1]
```diff
@@ -1075,3 +1075,33 @@
...
+  metadata:
+    - schedule
+    - tier-2
+    - openstack-only
+    - dmfg
+    - stage-7
+
+- name: "test to validate dashboard alerts"
+  suite: "suites/pacific/dashboard/tier-1_dashboard_alerts.yaml"
+  global-conf: "conf/pacific/cephadm/tier-1_3node_cephadm_bootstrap.yaml"
```
Comment (sunilkumarn417) [inline pipeline/metadata/5.3.yaml:1095]: Hey @manasagowri and @sunilangadi2 ,

please check whether `conf/pacific/cephadm/tier-1_3node_cephadm_bootstrap.yaml` file exists?
I was not able to find this file in repository.

image

Comment (pranavprakash20) [reply pipeline/metadata/5.3.yaml:1095]: +1

Comment (sunilangadi2) [reply pipeline/metadata/5.3.yaml]: Sorry @sunilkumarn417 it got missed with numeric 5,
Thanks for the cross check.
updated the proper conf file which exist in ceph-ci
Test results for the same conf:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-QZVCNK/
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-BNY9NS

**Review by pranavprakash20 — DISMISSED** [review]
LGTM

**Review by vamahaja — DISMISSED** [review]

**Comment by sunilangadi2** [comment]
@psathyan It's done resolved merge conflicts, please take a review.

**Review by psathyan — APPROVED** [review]

**Pull request merged**